src/extractor/extraction_containers.cpp

```diff
@@ -1224,7 +1232,7 @@ ExtractionContainers::ReferencedTrafficSignals ExtractionContainers::IdentifyTra
     util::for_each_indexed(ways_list.cbegin(), ways_list.cend(), set_segments);
 
     util::for_each_pair(
-        signal_segments, [](const auto pair_a, const auto pair_b) {
+        node_segments, [](const auto pair_a, const auto pair_b) {
```

Comment:
Minor: Warning message below could be updated to refer to traffic control flow nodes.